apps/condo/domains/common/components/TrackingContext.tsx

```diff
@@ -76,9 +116,8 @@ interface IUseTracking {
 }
 
 const useTracking: IUseTracking = () => {
-    const router = useRouter()
-    const route = get(router, 'route', '')
     const { trackerInstances, eventProperties, userProperties } = useTrackingContext()
+    const { route } = useRouter()
```

Comment:
Is `route` be defined on SSR? If not it could be runtime error with splitting `undefined`

Reply:
<img width="296" alt="image" src="https://user-images.githubusercontent.com/50069872/203936294-dd7e4263-ac41-4d0e-bdf2-6e134359c705.png">

So, next typing says, that route is always defined :)
And it's a bit strange to expect it to be null/undefined in SSR from SSR framework 😄